src/platform-includes/configuration/capture-console/javascript.mdx

```diff
@@ -22,11 +22,12 @@ Sentry.init({
 ></script>
 
 <script>
-  Sentry.onLoad(function () {
-    Sentry.init({
-      integrations: [new Sentry.Integrations.CaptureConsole()],
+  Sentry &&
```

Comment:
I feel like we should also say in a comment or something why we first check for `Sentry`'s existence:

eg `Check for existence of Sentry in case Ad-blockers block the Sentry Loader Script`

Reply:
Hmm I was thinking about this, my thought was it is a lot of repetition for all the integrations - but better safe than sorry I guess! I'll add it.